src/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+add_library(DLAF_Fortran dlaf_fortran.f90)
+
+target_compile_options(DLAF_Fortran PRIVATE -Wall -Wextra -Wpedantic)
+
+target_link_libraries(
+  DLAF_Fortran
+  PRIVATE DLAF::DLAF
```

Comment:
Shouldn't also `MPI::Fortran` be a dependency of `DLAF_Fortran`? If we agree on this, we should probably remove the "explicit" dependency elsewhere.

Reply:
After some discussion, we decided to leave it like this since the interface itself does not depend directly from MPI.